client/deposits/instant-deposits/index.js

```diff
@@ -4,28 +4,39 @@
  * External dependencies
  */
 import { Button } from '@wordpress/components';
-import { __ } from '@wordpress/i18n';
+import { __, _x, sprintf } from '@wordpress/i18n';
 import { useState } from '@wordpress/element';
+import moment from 'moment';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
 import InstantDepositModal from './modal';
 import { useInstantDeposit } from 'wcpay/data';
+import { formatCurrency } from 'utils/currency';
+import Tooltip from 'wcpay/components/tooltip';
+import { formatDate } from 'utils';
 
-const isButtonDisabled = ( instantBalance ) => {
+const isButtonDisabled = ( instantBalance, lastManualDeposit ) => {
```

Comment:
There is an opportunity to improve this by using a destructured object param (IMO):

```suggestion
const isButtonDisabled = ( { instantBalance, lastManualDeposit } ) => {
```